test/utility/misc.c

```diff
@@ -405,13 +405,32 @@ void
 testutil_wiredtiger_open(TEST_OPTS *opts, const char *home, const char *config,
   WT_EVENT_HANDLER *event_handler, WT_CONNECTION **connectionp, bool rerun, bool benchmarkrun)
 {
+    char auth_token[256], bucket_name[256];
     char buf[1024], tiered_ext_cfg[512];
+    const char *access_key, *secret_key;
 
-    if (opts->tiered_storage)
+    auth_token[0] = '\0';
+    if (opts->tiered_storage) {
+        strcpy(bucket_name,
+          strcmp(opts->tiered_storage_source, DIR_STORE) == 0 ? DIR_STORE_BUCKET_NAME :
+                                                                S3_STORE_BUCKET_NAME);
```

Comment:
Try using `getenv("WT_S3_EXT_BUCKET")` instead of `S3_STORE_BUCKET_NAME` as mentioned in another comment.

Reply:
Fixed in 5dc54e204facff54ad4cadf8cb4524b9e8dc03e7